src/styles/cards.scss

```diff
@@ -0,0 +1,101 @@
+.card {
```

Comment:
9'th point in checklist, file should be named exactly like the block it contains